pom.xml

```diff
@@ -7,8 +7,11 @@
         <!-- which version of Jenkins is this plugin built against? -->
     </parent>
 
+    <!-- Unique ID for the plugin.  Do not change this. -->
     <artifactId>ease-plugin</artifactId>
-    <version>1.2.13-SNAPSHOT</version>
+    <!-- Name that displays on the 'Manage Plugins' page of Jenkins. -->
+    <name>Arxan MAM Publisher</name>
```

Comment:
I'm guessing this works because artifactId is never visible to the user?

Reply:
It's displayed by default if you don't have a 'name' tag.  Adding the 'name' tag makes it invisible to the user.